### PR TITLE
Invert filter condition

### PR DESCRIPTION
**Pull request by ogkuzmin**
### Description

Fix filtering condition

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] Changelog updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [x] Reviewers added

### CODE REVIEW

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/HiddenMessageListItemFilter.kt [hunk 1]
```diff
@@ -15,6 +15,6 @@ internal object HiddenMessageListItemFilter : MessageListView.MessageListItemFil
     }
 
     override fun predicate(item: MessageListItem): Boolean {
-        return theirDeletedMessagePredicate.invoke(item).and(theirGiphyEphemeralMessagePredicate.invoke(item))
+        return theirDeletedMessagePredicate.invoke(item).and(theirGiphyEphemeralMessagePredicate.invoke(item)).not()
```
Comment (zsmb13) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/HiddenMessageListItemFilter.kt:18]: Any chance that we can restructure the code here to make it more readable? I get that inverting the value fixes it now, but the issue is that this line is super hard to understand in the first place.

**Review by bychkovdmitry — APPROVED** [review]

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/HiddenMessageListItemFilter.kt (hunk 1 above)
Comment (ogkuzmin) [reply stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/HiddenMessageListItemFilter.kt]: I refactored the solution. Please, have a look

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/HiddenMessageListItemPredicate.kt [hunk 2]
```diff
@@ -15,6 +15,6 @@ internal object HiddenMessageListItemFilter : MessageListView.MessageListItemFil
     }
 
     override fun predicate(item: MessageListItem): Boolean {
```
Comment (bychkovdmitry) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/HiddenMessageListItemPredicate.kt:17]: Will the second part be executed for giphy messages? Looks like a short circuit operation which succeeds after execution of the first part

**Review by bychkovdmitry — APPROVED** [review]

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/HiddenMessageListItemPredicate.kt (hunk 2 above)
Comment (ogkuzmin) [reply stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/HiddenMessageListItemPredicate.kt:17]: Yeah, should be conjunction

**Pull request merged**